requirements/constraints.txt

```diff
@@ -131,8 +131,3 @@ click==8.1.6
 
 # plz upgrade this in separate ticket
 redis==4.6.0
-
-# openedx-learning new version has some changes which are breaking quality tests
-# See https://github.com/openedx/openedx-learning/pull/68 for the changes.
-# It needs to be updated in a separate issue.
-openedx-learning==0.1.2
```

Comment:
@bradenmacdonald I removed this to make the tests pass. I'm not sure what "breaking quality tests" means here.

Reply:
@rpenido I believe that was because when my changes were merged to the `openedx-learning` repo, it contained changes that remove some kwargs from some functions that were being called in this repo, and that would cause the quality tests to fail (hence the version was pinned to not use the latest). Now that everything is being merged, this change should be fine, in fact, it is needed.

Reply:
Thank you @yusuf-musleh!